### PR TITLE
azure-security-nspkg

### PR DESCRIPTION
**Pull request by lmazuel**

### CODE REVIEW

File: sdk/security/azure-security-nspkg/azure/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+__path__ = __import__('pkgutil').extend_path(__path__, __name__)
```
Comment (johanste) [inline sdk/security/azure-security-nspkg/azure/__init__.py:1]: Is this here to make editable installs work (for python 2)? It is not included in the actual package (wheel/sdist), correct?

**Review by johanste — DISMISSED** [review]

File: sdk/security/azure-security-nspkg/azure/__init__.py (hunk 1 above)
Comment (lmazuel) [reply sdk/security/azure-security-nspkg/azure/__init__.py:1]: It's not in the wheel, but it's in the sdist. This applies to all packages (nspkg or not), precisely to simplify dev indeed.

Comment (lmazuel) [reply sdk/security/azure-security-nspkg/azure/__init__.py:1]: And I actually opened both wheel and sdist before doing the PR to check that was the case :p

**Comment by lmazuel** [comment]
Merging, since there is 0.000% chance that adding a nspkg package in security breaks core or anything else :p

**Pull request merged**